techtalk_slides.tex

```diff
@@ -329,6 +344,7 @@ \section{Irregular Verbs}
 \end{frame}
 
 \begin{frame}{Best strategy}
+    %  TODO: these slides are too wordy, simplify
```

Comment:
Yeah @mtphamine I wasn't sure exactly what changes would accomplish this, but these two slides feel a little too "just read everything off the slide".